plugins/woocommerce-admin/client/core-profiler/services/country.ts

```diff
@@ -0,0 +1,107 @@
+/**
+ * External dependencies
+ */
+import { decodeEntities } from '@wordpress/html-entities';
+
+function decodeHtml( content: string ) {
+	if ( ! content ) {
+		return '';
+	}
+
+	return decodeEntities( content );
+}
+/**
+ * Type definitions
+ */
+export type Country = {
```

Comment:
I think we can just use these types https://github.com/woocommerce/woocommerce/blob/8d3965d4ef41a9448ce1627b69680fe44b8c4077/packages/js/data/src/countries/types.ts#L24

Reply:
Updated in https://github.com/woocommerce/woocommerce/pull/38019/commits/f4d7698224ba58259f7f7ef0abf951682aacc79b